TYPO3.Flow/Classes/TYPO3/Flow/Property/TypeConverter/DateTimeConverter.php

```diff
@@ -138,12 +139,14 @@ public function convertFrom($source, $targetType, array $convertedChildPropertie
             }
             if (isset($source['dateFormat']) && strlen($source['dateFormat']) > 0) {
                 $dateFormat = $source['dateFormat'];
+                $isFormatSpecified = true;
             }
         }
         if ($dateAsString === '') {
             return null;
         }
-        if (ctype_digit($dateAsString) && $configuration === null && (!is_array($source) || !isset($source['dateFormat']))) {
+        $isFormatConfigured = ($configuration !== null && $configuration->getConfigurationValue(DateTimeConverter::class, self::CONFIGURATION_DATE_FORMAT) !== null);
+        if (!$isFormatConfigured && !$isFormatSpecified && ctype_digit($dateAsString)) {
```

Comment:
Is that enough to decide it's format 'U'?  I am not sure...
It could as well be YYMMDD, which is an allowed ISO notation (19800901 -> 1st September 1980), so IMHO you NEED to define the format.